library/modules/common/image/scss/error.scss

```diff
@@ -9,15 +9,15 @@
 			align-items: center;
 			justify-content: center;
 			flex-direction: column;
-			z-index: 2;
-			.pui-icon-button {
+			height: 100%;
+			.pragmate-icon-button {
```

Comment:
@FranciscoSanvicente the classname must be pui- not pragmate-ui.
Can you solve this ?